README.Rmd

```diff
@@ -63,13 +63,39 @@ matched <- match_name(loanbook_demo, ald_demo) %>%
   prioritize()
 ```
 
+### Add Scenario Targets
 * Use `target_sda()` to calculate SDA targets of CO2 emissions.
 
 ```{r}
-target_sda(matched, ald_demo, co2_intensity_scenario_demo)
+target_sda(matched, 
+           ald = ald_demo, 
+           co2_intensity_scenario = co2_intensity_scenario_demo)
 ```
 
-* Use `join_ald_scenario()` to join the matched dataset to the relevant 
+* Use `target_market_share` to calculate market-share scenario targets at the portfolio level:
+
+```{r}
+target_market_share(matched, 
```

Comment:
I'll run a styler. The tidyverse style guide reserves this style for function definitions (not calls), and only when they are longer than 80 characters.

For function calls the recommended style is this:

```r
foo(
  arg_1,
  more,
  arg_n
)
```

Reply:
Styled